Source/AdMobAdapterBannerAd.swift

```diff
@@ -66,7 +72,18 @@ extension AdMobAdapterBannerAd: GADBannerViewDelegate {
 
     func bannerViewDidReceiveAd(_ bannerView: GADBannerView) {
         log(.loadSucceeded)
-        loadCompletion?(.success([:])) ?? log(.loadResultIgnored)
+
+        // From https://developers.google.com/admob/ios/api/reference/Functions:
+        // "The exact size of the ad returned is passed through the banner’s ad size delegate and
+        // is indicated by the banner’s intrinsicContentSize."
+        let loadedSize = bannerView.intrinsicContentSize
+        var partnerDetails: [String: String] = [:]
+        partnerDetails["bannerWidth"] = "\(loadedSize.width)"
+        partnerDetails["bannerHeight"] = "\(loadedSize.height)"
+        // 0 for fixed size banner, 1 for adaptive banner.
+        partnerDetails["bannerType"] = (request.format == .banner ? "0" : "1")
```

Comment:
nit: this dictionary could be a let, without an explicit type

Reply:
Updated.